frontstage/views/surveys/help/surveys_help.py

```diff
@@ -187,13 +234,11 @@ def _send_new_message(subject, party_id, survey, business_id):
     return response
 
 
-def get_subject_and_breadcrumbs_title(option, uri):
-    """Gets the subject line for the secure message, the title of the breadcrumbs for sub options """
-    if option == 'do-not-have-specific-figures':
-        return 'I don’t have specific figures for a response', \
-               "Help completing this survey", \
-               "I don’t have specific figures for a response"
-    if option == 'unable-to-return-by-deadline':
-        return 'I’m unable to return the data by the deadline', \
-               "Help completing this survey", \
-               "I’m unable to return the data by the deadline"
+def _is_inside_legal_basis(legal_basis):
+    """
+    Returns true if the legal basis matches as mandatory.
+    :param: (str) the legal basis reference
```

Comment:
This still isn't quite right.  For an example you can look at the controllers (case_controller for a reasonable example).
This whole block should be:
```
"""
Checks whether the provided legal basis is for a survey which is mandatory.

:param legal_basis: The legal basis reference
:type legal_basis: str
:return: True if mandatory, false otherwise
:rtype: bool
"""
```

Reply:
done